theme/print.less

```diff
@@ -97,20 +101,20 @@
     }
 
     /** blockly **/
-    .blocklyPath {
+    .blocklyPath:not(.print-color) {
```

Comment:
can we do a document level ``:not(.print-color) .blockPath`` so we don't have to add/remove .print-color to individual items?

Reply:
I tried in different level, but it doesn't work :( The only way I could make the :not() selector work was in the element level itself

Reply:
Let me try to add new rules to print.less

Reply:
I was testing and it might work :)

Reply:
This one works better. Please check it out